<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="2">
     <widget class="QTextEdit" name="textEdit_64">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QTextEdit" name="textEdit_128">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QTextEdit" name="textEdit_2048">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>16:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3" colspan="2">
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>64:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3" colspan="2">
     <widget class="QLabel" name="label_64">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>find</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QSpinBox" name="spinBox">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>2048</number>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>128:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3" colspan="2">
     <widget class="QLabel" name="label_128">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>2048:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3" colspan="2">
     <widget class="QLabel" name="label_2048">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="5">
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>reset</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QTextEdit" name="textEdit_16">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
